.changelog/17582.txt

```diff
@@ -0,0 +1,5 @@
+```release-note:feature
+improved consul operator raft list-peers command added CommitIndex as well as Trails Leader By in the output table
+Commit Index is the LastIndex from /v1/operator/autopilot/health and Trails Leader By is the difference from leader
+last index and followers last index
+```
```

Comment:
Maybe short and sweet would be preferable here given the audience are customers interested in changes and how they benefit/affect them.

e.g. `raft list-peers` shows the number of commits each follower is trailing the leader by to aid in troubleshooting.